packages/siwe-parser/lib/abnf.ts

```diff
@@ -143,7 +144,7 @@ DIGIT          =  %x30-39
 HEXDIG         =  DIGIT / "A" / "B" / "C" / "D" / "E" / "F"
 `;
 
-export class ParsedMessage {
+export class ABNFParsedMessage {
```

Comment:
Do we want to rename these? If we rename them, we need to bump a major version for `siwe-parser` otherwise it's a minor version change.